toolz/itertoolz.py

```diff
@@ -214,7 +214,7 @@ def _merge_sorted_binary_key(seqs, key):
         yield val1
 
 
-def interleave(seqs, pass_exceptions=()):
+def interleave(seqs):
```

Comment:
Well, it didn't appear to be documented or tested, so hopefully backwards compatibility isn't an issue.

Also, I don't get the use case.  Iterables must support the iteration protocol.  If custom substitutes for `StopIteration` are supported, wouldn't that affect everything in `itertoolz`?